### ISSUE TITLE
Hint Content Discussion

### ISSUE THREAD

**Issue by alexhude** [alexhude/FRIEND#1]
Personally, I prefer to keep hints small and display only important information there. Therefore we need to decide what should be considered as **important**.

For now it is:
1. **Purpose description** (what is this element for/doing)
2. **Format for instructions** (what data is passed)
3. **Bitfields for registers** (what values are set)

I am open to other suggestions :)

**Comment by devnoname120** [comment]
I'm not very familiar with IDA's API but would it be possible to make the hints scrollable like IDA's code sneak peek boxes?

This way short and important information could be displayed, and scrolling would bring extended information.

**Comment by alexhude** [comment]
No, unfortunately IDA hints are not scrollable. And I think it is fine because what we actually want to see is a brief info about instruction/register. There is "Show Documentation" item in popup menu for full description.

**Comment by soloveri** [comment]
hello,there are  some errors when I built the FRIEND on windows10 64bit by VS2017, it said that could not open some files contains idp.hpp、ida.hpp and loader.hpp when built the FRIEND.dll and FRIEND64.dll  because no such file or directory .Can you give me some advice? thank you

**Comment by alexhude** [comment]
I would recommend to check path to your IDA SDK.
P.S. I will have to remove these messages later since they clearly don't fit "Hint Content Discussion" topic :)
